### PR TITLE
testing pull

### PR DESCRIPTION
**Pull request by AditiSurana**
Added div, please pull

### CODE REVIEW

**Comment by AditiSurana** [comment]
Please merge this

File: index.html [hunk 1]
```diff
@@ -159,6 +161,7 @@
   <div id ="step2">
   </div>
   <div id ="step3">
+    <div> Hi </div>
```
Comment (AditiSurana) [inline index.html:164]: I think we should add Hi

**Review by AditiSurana — COMMENTED** [review]
Review 1

File: index.html (hunk 1 above)
Comment (sings257) [reply index.html:164]: Okay. #1

**Review by sings257 — APPROVED** [review]
Okay let's do it.

**Pull request merged**